ban/core/exceptions.py

```diff
@@ -1,3 +1,7 @@
+class ValidationError(ValueError):
+    ...
```

Comment:
Heh, I discovered that too recently, still wondering if that's more explicit than the usual `pass`.